domain/src/main/java/org/oppia/android/domain/topic/StoryProgressTestHelper.kt

```diff
@@ -24,7 +24,8 @@ class StoryProgressTestHelper @Inject constructor(
    * Creates a partial story progress for a particular profile.
    *
    * @param profileId The profile we are setting partial progress of the fraction story for.
-   * @param timestampOlderThanOneWeek If the timestamp for this topic progress is more than one week ago.
+   * @param timestampOlderThanOneWeek If the timestamp for this topic progress is more
+   * than one week ago.
```

Comment:
```suggestion
   *     than one week ago.
```

We should be using four spaces to indent continuations in KDocs (https://developer.android.com/kotlin/style-guide#block_tags).

Ditto below.

Reply:
Done.